From 0283d199517f8c0b798af1a1405cc5fb9999d11e Mon Sep 17 00:00:00 2001
From: Mahesh Bodapati <mbodapat@xilinx.com>
Date: Mon, 23 Jan 2017 15:30:02 +0530
Subject: [PATCH 01/11] Modified _exceptional_handler Modified the
 _exceptional_handler to support the changes made in GCC related to
 Superviosry call

Signed-off-by:Nagaraju Mekala<nmekala@xilix.com>

Signed-off-by: Aayush Misra <aayushm@amd.com>
Upstream-Status: Inappropriate [Xilinx local]
---
 libgloss/microblaze/_exception_handler.S | 1 -
 1 file changed, 1 deletion(-)

diff --git a/libgloss/microblaze/_exception_handler.S b/libgloss/microblaze/_exception_handler.S
index 59385ad9b..7a91a781e 100644
--- a/libgloss/microblaze/_exception_handler.S
+++ b/libgloss/microblaze/_exception_handler.S
@@ -36,5 +36,4 @@
 	.type _exception_handler, @function
 	
 	_exception_handler:
-	addi r11,r11,8
 	bra r11
-- 
2.43.7


